registry-disk.md

```diff
@@ -0,0 +1,77 @@
+## Registry Disk
+
+The Registry Disk feature provides the ability to store and distribute Virtual
+Machine disks in the container image registry. Registry Disks can be assigned
+to Virtual Machines in the disks section of the Virtual Machine spec.
+
+## When to use a Registry Disk
+
+Registry Disks are ephemeral storage devices that can be assigned to any number
```

Comment:
I think it is clear, if one thinks about the container registry re-use, but could you add one or two sentences, which make it more explicit, that no shared storage is needed, after the images are pulled?

Reply:
i'll add some more about this